### PR TITLE
Add surface loss in keras

### PR DESCRIPTION
**Pull request by akamojo**
Together with @marcinkaczor, we added a simple script which includes functions for calculating surface loss using keras. Feel free to modify it according to your needs.

### CODE REVIEW

**Comment by HKervadec** [comment]
Great ! Thanks for the contribution.

I will review soon (will be mostly in a plane for the next few days), I will think about the best way to integrate it in the current repository. Modifying from binary to multi-class should be trivial also, but perhaps I will end up with two separate functions.
